backend/config/settings/local.py

```diff
@@ -3,7 +3,18 @@
 # Dummy value for development
 SECRET_KEY = "*m(r@4mdh*!zabwg&6tp%mgs_ezkprs9g+$@x@cdq-z_)dtf2i"
 
-DEBUG_TOOLBAR = False
+DEBUG_TOOLBAR = True
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
```

Comment:
This will replace the default sqlite, which automatically creates a db. The setup part of readme should now be updated to mention that PostgreSQL is used by default and while rest of the stack runs locally, run `docker-compose -f services.yml up -d` to get the db running through docker.

Reply:
I think I don't have access to that part!

Reply:
I think you mean that you can't edit it directly in browser? It should be possible to edit the readme and include it as changes part of the PR.